controllers/util/util.go

```diff
@@ -50,6 +50,14 @@ func GetInstallScope() string {
 	return ns
 }
 
+func GetOdlmScope() string {
```

Comment:
any reason why this function doesn't return a boolean instead?

Reply:
Great point, updated!